server/config/config.go

```diff
@@ -75,6 +77,7 @@ func DefaultConfig() *Config {
 		BaseConfig{
 			MinGasPrices:    defaultMinGasPrices,
 			InterBlockCache: true,
+			Pruning:         "",
```

Comment:
This should be inline with the default CLI value.

```suggestion
			Pruning:         "syncable",
```